pages/index.tsx

```diff
@@ -11,11 +11,16 @@ import { getRandomIcon } from "utils/get-random-icon";
 import { persistEmoji, getStoredEmoji } from "utils/storage";
 
 import styles from "styles/Home.module.css";
+import Link from "next/link";
```

Comment:
this import should be with the third-party ones

Reply:
Done